apps/easypid/src/features/share/FunkeOpenIdPresentationNotificationScreen.tsx

```diff
@@ -102,10 +101,10 @@ export function FunkeOpenIdPresentationNotificationScreen() {
   const usePin = useMemo(() => {
     const isPidInSubmission =
       submission?.entries.some((entry) =>
-        entry.credentials.some((credential) => credential.id === pidCredential?.id)
```

Comment:
@janrtvld i think the submission credentials should also just use the credentials for display. And I was thinking of adding a `isPid` to the credential for display. That way we can just access that